README.md

```diff
@@ -98,6 +98,37 @@ variables for the test scripts like so:
 
 > `$ make DOCKER_OPTS="-e TEST_ALL_RUBIES=1" test`
 
+In order to run the tests manually in the container-based development, the services
+needed to run them correctly must be started before with:
+
+> `$ script/services start`
+
+Then you can execute the following commands to execute all tests:
+
+> `$ bundle exec rake`
+
+Another alternative to execute all tests from within the development environment
+where it is not needed to manually start the services before is:
```

Comment:
this is not correct: `it is not needed` should be replaced with `you should not` (ie. it might work only out of chance and it is likely to produce problems) - and maybe mention how to stop the services in case you had them running.